admin/templates/util/pagination.html

```diff
@@ -3,7 +3,7 @@
     <div class="pagination pagination-lg">
         <span>
             {% if items.has_previous %}
-                <a href="?page={{ items.previous_page_number }}"
+                <a href="?page={{ items.previous_page_number }}&status={{ status }}"
```

Comment:
I think this is best left for when we look into refactoring the prereg into a CBV rather than FBV. Definitely a worthwhile consideration.